towhee/hub/file_manager.py

```diff
@@ -280,7 +278,7 @@ def _cache_locals(self):
                     # TODO: Figure out exception types
                     copytree(str(old_path), str(new_path))
 
-    def get_pipeline(self, pipeline: str, branch: str = 'main', redownload: bool = False, install_reqs=True):
+    def get_pipeline(self, pipeline: str, branch: str = 'main', install_reqs=True):
```

Comment:
We should change the name from `branch` to `tag`,  and the default value is `None`, which means the pipeline will use `main`.

Reply:
Will create another PR to change all the `branch` to `tag`